### PR TITLE
Update guide.md

### PR DESCRIPTION
**Pull request by testkil**
Clarifications, some ppl were confused why "requires tank skills" wasn't on there. + mentioning the bastion badge

### CODE REVIEW

File: frontend/src/Pages/Guide/guides/bastion/guide.md [hunk 1]
```diff
@@ -12,15 +12,16 @@ Bastion is for experienced incursion runners only. This means to use it you MUST
 4.  Know the different Sansha rats.
 5.  Be able to communicate with the FC.
 
-## General rules if you want SRP
+## General rules/requirements if you want SRP
```
Comment (TvdW) [inline frontend/src/Pages/Guide/guides/bastion/guide.md:15]: Doesn't a rule imply that it's a requirement?

File: frontend/src/Pages/Guide/guides/bastion/guide.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Bastion guide
 
-With the Bastions of War changes, marauders now do double damage while in Bastion. This is obviously huge, but comes at the downside of being unable to receive reps from logi. This guide should cover most things you need to know to run Bastion whilst receving SRP.
+With the Bastions of War changes, marauders now do double damage while in Bastion. This is obviously huge, but comes at the downside of being unable to receive reps from logi. This guide should cover most things you need to know to run Bastion whilst receving SRP. The bastion badge, which requires full elite + abyssal damage mods is not required to use bastion but it is encouraged to obtain it.
```
Comment (TvdW) [inline frontend/src/Pages/Guide/guides/bastion/guide.md]: Missing a comma for readability

File: frontend/src/Pages/Guide/guides/bastion/guide.md [hunk 3]
```diff
@@ -12,15 +12,16 @@ Bastion is for experienced incursion runners only. This means to use it you MUST
...
+## General rules/requirements if you want SRP
 
 1.  **Auto repeat is off at all times.**
 2.  **Bastion in lowest module row, NO SHORTCUT!**
 3.  You do not Bastion until aggro is established on one target at the start of a wave/site.
 4.  If someone dies on-grid you are to turn Bastion off for the rest of the site.
 5.  Full Hybrid or Amulet pod is required for SRP.
-6.  No abyssal plates on Bastion ships.
-7.  Be aware when using the micro jump drive of your direction, and of bumps. If you jump poorly and die due to being out of logi range, this is pilot error and WILL NOT be covered under SRP. (Being bumped by another fleet member is also not covered under SRP). If you're not confident, DO NOT USE IT.
+6.  Elite tank skills.
```
Comment (TvdW) [inline frontend/src/Pages/Guide/guides/bastion/guide.md]: Should this be specified to be more clear?

File: frontend/src/Pages/Guide/guides/bastion/guide.md (hunk 1 above)
Comment (testkil) [reply frontend/src/Pages/Guide/guides/bastion/guide.md:15]: Yeah, but that isn't always clear for non native speakers. We always talk about requirements and on this page we say rules.

File: frontend/src/Pages/Guide/guides/bastion/guide.md (hunk 2 above)
Comment (testkil) [reply frontend/src/Pages/Guide/guides/bastion/guide.md]: fixed

**Pull request merged**